packages/vue-pdf-viewer/src/pdf/PdfJsHelper.js

```diff
@@ -103,6 +103,14 @@ export class PdfJsHelper {
         return await getDocument(docOptions).promise
     }
 
+    /**
+     * @param {PDFDocumentProxy} pdfDocument
+     * @returns {Promise<Uint8Array>}
+     */
+    async getPdfData(pdfDocument) {
+        return await pdfDocument.saveDocument()
+    }
```

Comment:
I didn't add this, because if you have to pass the `pdfDocument` anyway, you can directly call `saveDocument`.

Reply:
Oh true :D Removed again